doc/source/writers.rst

```diff
@@ -54,6 +54,67 @@ To get a list of available writers use the `available_writers` function::
     >>> from satpy import available_writers
     >>> available_writers()
 
+
+Examples
+========
+
+CF-Writer
```

Comment:
Should these examples maybe be put in the python module docstring and then have this page point to the API docs?

Reply:
That makes sense! The closer they are to the code, the more likely they will be kept up-to-date